packages/govuk-frontend-review/src/app.puppeteer.test.mjs

```diff
@@ -0,0 +1,173 @@
+import {
+  getAttribute,
+  getProperty,
+  goTo
+} from '@govuk-frontend/helpers/puppeteer'
+import { getComponentNames } from '@govuk-frontend/lib/components'
+
+describe('Other examples', () => {
+  describe.each([
+    '/examples/form-alignment',
+    '/examples/form-elements',
+    '/examples/grid',
+    '/examples/links',
+    '/examples/typography',
+    '/examples/template-default',
+    '/examples/template-custom'
+  ])('%s', (path) => {
+    beforeAll(async () => {
+      await goTo(page, path)
+    })
+
+    it('should load correctly', async () => {
+      const $title = await page.$('title')
+
+      // Check the page responded correctly
+      expect(getProperty($title, 'textContent')).resolves.toContain('GOV.UK')
+    })
+  })
+
+  describe('/examples/template-custom', () => {
```

Comment:
I think these tests pre-date us (cccb7d3352da08ef03312709cf629992a567542c) introducing separate tests for the template itself (3095e165ebb5d2fc945711cbd9c5393ab040b0ec).

There does seem to be a lot of overlap, and these are really testing the template rather than the review app. I do wonder if we should just delete this whole `describe` block instead?

Happy to create an issue to review this later if you prefer.

Reply:
Let's remove it

On the basis that I removed all the HTTP header checks too

We already have [tests for `x-robots-tag`](https://github.com/alphagov/govuk-frontend/commit/4b1a3695f2eaac3f3ef6ed8448440281a988ee38#diff-ddb795bb152f353e17602ff9179ab6f6a97532b3608013f57accad032bb1c2d5) and Puppeteer will tell us should `text/html` go wrong

Reply:
Added an extra commit for the removal https://github.com/alphagov/govuk-frontend/pull/4205/commits/f26d60d06ec3442a76122a11cfe7073e6c8ebfd4